### PR TITLE
todo list using react

### PR DESCRIPTION
**Pull request by boo-shehab**
# This branch is feature branch for to-do-list application
- Setup react application
- Setup linters configuration
- Remove unnecessary files in react app
- Added Components folder
- Added some js files in components folder
- Created Add functionality for to-do-list
- Created Remove functionality for to-do-list
- Created Edit functionality for to-do-list
- Created Check functionality for to-do-list
- Added local-storage
- Used React Hooks
- Setup React-Router
- Added About Page
- Changed Readme file
- Added .gitignore file
- Fixed all linters errors

### CODE REVIEW

File: README.old.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# todo-list
```
Comment (usamamaliq) [inline README.old.md]: Kindly remove this readme file as it contains nothing.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
...
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+### `npm test`
+
```
Comment (usamamaliq) [inline README.md:18]: In my opinion, you should only follow the new template of the readme file and should remove the auto-generated readme description while creating react app.

File: src/components/TodoContainer.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+/* eslint-disable no-param-reassign */
+/* eslint-disable react/no-access-state-in-setstate */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/sort-comp */
+/* eslint-disable react/state-in-constructor */
```
Comment (usamamaliq) [inline src/components/TodoContainer.js]: Optional: I can see you have disabled ES-Linter a lot in multiple files. I know that you had less time to complete the application but try to fix the errors in your spare time as this is a clean code requirement.

**Review by usamamaliq — CHANGES_REQUESTED** [review]
@boo-shehab,

**Changes Required**

14bo


Good Job implementing the project requirements. There are some changes that I have proposed. Kindly fix those to move on to the next project. 👍
 
**Note: The changes marked optional are optional.**

**HIGHLIGHTS**

-  Your project is professional
-  Your repo is professional

**Review by usamamaliq — APPROVED** [review]
Hi @boo-shehab,

# Status: Approved ✔️ ✔️ ✔️

7MyJ


## Top Highlight 💪🏻

React Installed ✔️
React App opens in the browser ✔️
No Linters errors✔️
New Readme Template Used ✔️

Good job implementing the requirements 👍 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

**Congratulations!** 🎉

**Comment by boo-shehab** [comment]
thank you

**Pull request merged**